File: app-ui/lib/models/period.dart
class Period {
  DateTime _startDate;
  DateTime? _finishDate;

  DateTime get startDate => _startDate;
  DateTime? get finishDate => _finishDate;

  set startDate(DateTime startDate) => _startDate = startDate;
  set finishDate(DateTime? finishDate) => _finishDate = finishDate;

  Period(this._startDate, this._finishDate);
}

File: service-server/src/main/java/com/rocketdan/serviceserver/domain/event/element/Period.java
package com.rocketdan.serviceserver.domain.event.element;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Getter;

import java.util.Date;

@Getter
@AllArgsConstructor
public class Period {
    // 이벤트 영구 유지 여부
    private Boolean isPermanent;

    // 이벤트 시작 시간
    @JsonFormat(shape=JsonFormat.Shape.STRING, pattern="yyyy-MM-dd'T'HH:mm:ss")
    private Date startDate;

    // 이벤트 끝 시간
    @JsonFormat(shape=JsonFormat.Shape.STRING, pattern="yyyy-MM-dd'T'HH:mm:ss")
    private Date finishDate;
}